packages/openapi-generator/src/document-converter.ts

```diff
@@ -69,24 +71,60 @@ export async function convertDocToOpenApiV3(
 }
 
 /**
- * Workaround for OpenAPI generation to build one and only one API for all tags.
- * Modify spec to contain only one 'default' tag.
+ * Modify spec to contain the 'default' tag when no tags are defined.
  * @param openApiDocument OpenAPI JSON document.
  * @returns The modified document.
  */
-export function convertDocToGlobalTag(
+export function convertDocWithDefaultTag(
```

Comment:
@FrankEssenberger , regarding the change for the global tag handling.

Reply:
ah there it is thanks.